llvm/lib/Target/AMDGPU/SIISelLowering.cpp

```diff
@@ -53,9 +53,13 @@ using namespace llvm::SDPatternMatch;
 
 STATISTIC(NumTailCalls, "Number of tail calls");
 
+static cl::opt<bool>
+    DisableAllLoopAlignment("amdgpu-disable-all-loop-alignment",
+                            cl::desc("Do not align loops"), cl::init(false));
+
 static cl::opt<bool>
     DisableLoopAlignment("amdgpu-disable-loop-alignment",
-                         cl::desc("Do not align and prefetch loops"),
+                         cl::desc("Do not align loops for prefetch"),
                          cl::init(false));
```

Comment:
Can we not add new flags? There is a cost to them and we should be removing many of the ones we already have. Clang already has an explicit metadata driven flag to align or disable alignment of loops.